flow_action_components/ChatSessionManagerLA/ChatSessionManagerLA.cmp

```diff
@@ -0,0 +1,7 @@
+<aura:component implements="lightning:availableForFlowActions" access="global">
```

Comment:
Since this is a Local Action, I suggest you name it ManageChatSession instead of ChatSessionManagerLA. We like to name actions starting with a verb.

Reply:
I will create a new pull request with a new name for the action component : ManageChatSession